pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -360,7 +360,28 @@ def gen_object_with_cleanup():
 
     def _gen_object_with_cleanup(api_client, data):
         new_obj = api_client.create(data)
-        obj_hrefs.append((api_client, new_obj.pulp_href))
+        try:
+            obj_hrefs.append((api_client, new_obj.pulp_href))
+        except AttributeError:
+            # This is a task and the real object href comes from monitoring it
+            import pydevd_pycharm
+
+            pydevd_pycharm.settrace(
+                "localhost", port=29437, stdoutToServer=True, stderrToServer=True
+            )
```

Comment:
was this supposed to be here?

Reply:
Nope, and it need a bit more work too. I'm moving back to draft. Thanks for commenting on this.